src/applications/personalization/dashboard/components/ManageYourVAHealthCare.jsx

```diff
@@ -145,10 +145,11 @@ const ManageYourVAHealthCare = ({
     >
       <PrescriptionsWidget />
     </DowntimeNotification>
+    {(isEnrolledInHealthCare && !isCernerPatient) ||
+      (environment.isProduction() && <ScheduleAnAppointmentWidget />)}
```

Comment:
the grouping on this one doesn't look quite right. I think it should be
`{((isEnrolledInHealthCare && !isCernerPatient) || environment.isProduction()) && <ScheduleAnAppointmentWidget />}`
or maybe even a little clearer to reorganize it as:
`{(environment.isProduction() || (isEnrolledInHealthCare && !isCernerPatient)) && <ScheduleAnAppointmentWidget />}`